pkg/server/routes.go

```diff
@@ -33,9 +33,9 @@ func setupRoutes(cfg *Config, authChecker auth.Checker) *mux.Router {
 	api.HandleFunc("/loki/metrics", handler.LokiMetrics(&cfg.Loki))
 	api.HandleFunc("/loki/buildinfo", handler.LokiBuildInfos(&cfg.Loki))
 	api.HandleFunc("/loki/config/limits", handler.LokiConfig(&cfg.Loki, "limits_config"))
-	api.HandleFunc("/loki/flows", handler.GetFlows(&cfg.Loki))
+	api.HandleFunc("/loki/flow/records", handler.GetFlows(&cfg.Loki))
```

Comment:
+1 on that, good renaming 
that might break QE tests though, ping @memodi @Amoghrd just in case

Reply:
sure, added a mention in the description

Reply:
Thanks guys, I've opened a PR for updates here: https://github.com/openshift/openshift-tests-private/pull/11959